docs/cow-protocol/tutorials/cow-swap/swap.mdx

```diff
@@ -16,24 +16,28 @@ Consider an example swapping 0.05 `WETH` for `COW`.
 
 <Image img={require('/img/cowswap/swap_need_approval.png')} alt="GPv2VaultRelayer approval required" />
 
-When you do a swap on CoW Swap for the first time, you must give approval for the [`GPv2VaultRelayer`](/cow-protocol/reference/contracts/core#gpv2vaultrelayer) contract to spend your token (in this case, `WETH`). 
-This is because [`GPv2VaultRelayer`](/cow-protocol/reference/contracts/core#gpv2vaultrelayer) is the contract that will pull the `WETH` from our wallet and send it to the [settlement](/cow-protocol/reference/contracts/core#gpv2settlement) contract that will co-ordinate the swap on your behalf.
-Both contracts are audited, battle-tested and non-upgradeable.
-Moreover, the settlement contract is the only address that can request the relayer to pull your funds and only after verifying both that your order was signed by you and that you will receive at least your limit execution price or better. 
+When you do a swap on CoW Swap for the first time, you must give approval for the [Vault relayer contract](/cow-protocol/reference/contracts/core/vault-relayer) contract to spend your token (in this case, `WETH`). 
+
+This is because this contract needs to pull the `WETH` from our wallet and send it to the [Settlement contract](/cow-protocol/reference/contracts/core/settlement) that will co-ordinate the swap on your behalf.
```

Comment:
```suggestion
This is because this contract needs to pull the `WETH` from our wallet and send it to the [_settlement_](/cow-protocol/reference/contracts/core/settlement) contract that will co-ordinate the swap on your behalf.
```